vm_method.c

```diff
@@ -122,6 +122,7 @@ vm_cme_invalidate(rb_callable_method_entry_t *cme)
     RB_DEBUG_COUNTER_INC(cc_cme_invalidate);
 
     rb_yjit_cme_invalidate(cme);
+    rb_zjit_cme_invalidate(cme);
```

Comment:
There were other existing `rb_yjit_cme_invalidate` calls, and they should have had `rb_zjit_cme_invalidate` too. I'll add that in https://github.com/ruby/ruby/pull/14376 that happens to trigger the problem.

Reply:
Ah good catch and sorry 🥲